opencsp/common/lib/render/figure_management.py

```diff
@@ -104,8 +134,8 @@ def tile_figure(
     ul_y = size_y_pixels * y_idx
 
     # Create figure.
-    # fig = plt.figure(constrained_layout=True).subplots(5, 5)
-    fig = plt.figure(name, figsize=(size_x, plot_size_y))
+    # fig = figure(constrained_layout=True).subplots(5, 5)

```

Comment:
If you think it's a good idea, could we remove this commend out line? For the sake of preserving what we know works and minimizing the likelihood of someone accidently bringing up this error again.

Reply:
That's OK with me. @bbean23 what do you think?

Reply:
I think it'd be good to remove the commented out line. Note that the `figure()` method name changed again in #15, so you should probably pull again.